secp256k1jni/src/main/java/org/bitcoin/Secp256k1Context.java

```diff
@@ -46,7 +48,12 @@
   }
 
   public static boolean isEnabled() {
-     return enabled;
+      String secpDisabled = System.getenv("DISABLE_SECP256K1");
+      if (secpDisabled != null && secpDisabled.equals("true")) {
```

Comment:
probably should convert `secpDisabled` to lowercase

Reply:
maybe allow it to be `1` as well